<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RequestAuthorization</class>
 <widget class="QDialog" name="RequestAuthorization">
  <property name="windowModality">
   <enum>Qt::WindowModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>809</width>
    <height>373</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>330</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Request For Authorization</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="2" column="0" colspan="6">
    <widget class="Line" name="line">
     <property name="lineWidth">
      <number>2</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="6" column="3">
    <widget class="QCheckBox" name="extend_public_key">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>50</height>
      </size>
     </property>
     <property name="text">
      <string>Exchange of extended public keys</string>
     </property>
    </widget>
   </item>
   <item row="8" column="0">
    <widget class="QLabel" name="label_message">
     <property name="text">
      <string>Message</string>
     </property>
    </widget>
   </item>
   <item row="10" column="1">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>28</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="9" column="0" colspan="6">
    <widget class="QTextEdit" name="message">
     <property name="html">
      <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MS Shell Dlg 2'; font-size:8.25pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;a name=&quot;result_box&quot;&gt;&lt;/a&gt;&lt;span style=&quot; font-size:8pt;&quot;&gt;H&lt;/span&gt;&lt;span style=&quot; font-size:8pt;&quot;&gt;ello, I'd like to add you as an authorized contact.&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
   </item>
   <item row="11" column="0" colspan="4">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>265</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="11" column="4">
    <widget class="QPushButton" name="button_send">
     <property name="text">
      <string>Send</string>
     </property>
    </widget>
   </item>
   <item row="11" column="5">
    <widget class="QPushButton" name="button_cancel">
     <property name="text">
      <string>Cancel</string>
     </property>
    </widget>
   </item>
   <item row="7" column="0" colspan="6">
    <widget class="QGroupBox" name="add_contact">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>50</height>
      </size>
     </property>
     <property name="title">
      <string>Add as New Contact</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
     <widget class="QLineEdit" name="first_name">
      <property name="geometry">
       <rect>
        <x>123</x>
        <y>20</y>
        <width>181</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>20</y>
        <width>81</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>First Name</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>404</x>
        <y>20</y>
        <width>81</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Last Name</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="last_name">
      <property name="geometry">
       <rect>
        <x>486</x>
        <y>20</y>
        <width>181</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
     </widget>
    </widget>
   </item>
   <item row="6" column="0" colspan="2">
    <widget class="QGroupBox" name="direct_connect">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>50</height>
      </size>
     </property>
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
     <property name="title">
      <string>Direct Connect</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
     <property name="flat">
      <bool>false</bool>
     </property>
     <property name="checkable">
      <bool>false</bool>
     </property>
     <widget class="QCheckBox" name="check_box_chat">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>20</y>
        <width>70</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Chat</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QCheckBox" name="check_box_mail">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>20</y>
        <width>70</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Mail</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
    </widget>
   </item>
   <item row="5" column="0" colspan="6">
    <widget class="KeyhoteeIDPubKeyWidget" name="keyhoteeidpubkey" native="true">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>72</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="0" column="0" colspan="5">
    <widget class="IdentitySelection" name="widget_Identity" native="true">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>25</height>
      </size>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KeyhoteeIDPubKeyWidget</class>
   <extends>QWidget</extends>
   <header>AddressBook/keyhoteeidpubkeywidget.hpp</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>IdentitySelection</class>
   <extends>QWidget</extends>
   <header>Identity/IdentitySelection.hpp</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>check_box_chat</tabstop>
  <tabstop>check_box_mail</tabstop>
  <tabstop>extend_public_key</tabstop>
  <tabstop>add_contact</tabstop>
  <tabstop>first_name</tabstop>
  <tabstop>last_name</tabstop>
  <tabstop>message</tabstop>
  <tabstop>button_send</tabstop>
  <tabstop>button_cancel</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>button_cancel</sender>
   <signal>released()</signal>
   <receiver>RequestAuthorization</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>416</x>
     <y>199</y>
    </hint>
    <hint type="destinationlabel">
     <x>233</x>
     <y>109</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>button_send</sender>
   <signal>released()</signal>
   <receiver>RequestAuthorization</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>325</x>
     <y>199</y>
    </hint>
    <hint type="destinationlabel">
     <x>233</x>
     <y>109</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
